config/jobs/mirror-images/periodics-mirror-images.yaml

```diff
@@ -0,0 +1,25 @@
+periodics:
```

Comment:
I think we can run this job as a post-submit when `source-images.yaml` is touched.